dist/docsite/js/cdr-tokens.cjs

```diff
@@ -1,90 +1,90 @@
+  "CdrColorTextInputOptional": "rgba(75, 74, 72, 0.75)",
+  "CdrColorTextInputDisabled": "#d5cfc3",
+  "CdrColorTextInputFilled": "#2e2e2b",
+  "CdrColorTextInputHelp": "rgba(75, 74, 72, 0.75)",
   "CdrColorTextInputError": "#b33322",
-  "CdrColorTextLinkRest": "rgba(3, 3, 1, 0.9)",
-  "CdrColorTextLinkHover": "rgba(3, 3, 1, 0.9)",
-  "CdrColorTextLinkActive": "rgba(12, 11, 8, 0.75)",
-  "CdrColorTextLinkDisabled": "#d1cbbd",
+  "CdrColorTextLinkRest": "rgba(46, 46, 43, 0.9)",
+  "CdrColorTextLinkHover": "rgba(46, 46, 43, 0.9)",
+  "CdrColorTextLinkActive": "rgba(46, 46, 43, 0.75)",
+  "CdrColorTextLinkDisabled": "#d5cfc3",
   "CdrColorTextLinkVisited": "#406eb5",
-  "CdrColorTextRatingDefault": "rgba(12, 11, 8, 0.75)",
+  "CdrColorTextRatingDefault": "rgba(46, 46, 43, 0.75)",
   "CdrColorTextRatingHover": "#406eb5",
   "CdrColorTextRatingSeparator": "#b2ab9f",
-  "CdrColorTextTabRest": "rgba(66, 59, 47, 0.75)",
-  "CdrColorTextTabActive": "rgba(12, 11, 8, 0.75)",
+  "CdrColorTextTabRest": "rgba(75, 74, 72, 0.75)",
+  "CdrColorTextTabActive": "rgba(46, 46, 43, 0.75)",
   "CdrColorTextTabHover": "#406eb5",
-  "CdrColorTextTabDisabled": "#d1cbbd",
-  "CdrColorTextToggleButtonDefaultRest": "#4e4d49",
-  "CdrColorTextTooltipDefault": "#f9f8f6",
-  "CdrColorTextTablePrimary": "rgba(3, 3, 1, 0.9)",
-  "CdrColorTextTableSecondary": "rgba(12, 11, 8, 0.75)",
+  "CdrColorTextTabDisabled": "#d5cfc3",
+  "CdrColorTextToggleButtonDefaultRest": "#4b4a48",
+  "CdrColorTextTooltipDefault": "#fafbf9",
+  "CdrColorTextTablePrimary": "rgba(46, 46, 43, 0.9)",
+  "CdrColorTextTableSecondary": "rgba(46, 46, 43, 0.75)",
   "CdrColorTextUsageAffirmative": "#3b8349",
   "CdrColorTextUsageNegative": "#bb4045",
   "CdrColorTextUsageCaution": "#b68b37",
   "CdrColorBackgroundButtonPrimaryRest": "#1f513f",
   "CdrColorBackgroundButtonPrimaryActive": "#1f513f",
-  "CdrColorBackgroundButtonPrimaryHover": "#f4f2ed",
-  "CdrColorBackgroundButtonPrimaryIconAltHover": "#f9f8f6",
-  "CdrColorBackgroundButtonSecondaryRest": "#f9f8f6",
-  "CdrColorBackgroundButtonSecondaryActive": "#928b80",
-  "CdrColorBackgroundButtonSecondaryHover": "#f4f2ed",
-  "CdrColorBackgroundButtonSecondaryDisabled": "#f9f8f6",
-  "CdrColorBackgroundButtonDarkRest": "#4e4d49",
-  "CdrColorBackgroundButtonDarkActive": "#4e4d49",
+  "CdrColorBackgroundButtonPrimaryHover": "#f7f5f3",
+  "CdrColorBackgroundButtonPrimaryIconAltHover": "#fafbf9",
+  "CdrColorBackgroundButtonSecondaryRest": "#fafbf9",
+  "CdrColorBackgroundButtonSecondaryActive": "#958e83",
+  "CdrColorBackgroundButtonSecondaryHover": "#f7f5f3",
+  "CdrColorBackgroundButtonSecondaryDisabled": "#fafbf9",
+  "CdrColorBackgroundButtonDarkRest": "#4b4a48",
+  "CdrColorBackgroundButtonDarkActive": "#4b4a48",
   "CdrColorBackgroundButtonDarkHover": "#edeae3",
   "CdrColorBackgroundButtonSaleRest": "#c7370f",
   "CdrColorBackgroundButtonSaleActive": "#c7370f",
-  "CdrColorBackgroundButtonSaleHover": "#f4f2ed",
```

Comment:
these colors seem to be rather different 

<img width="194" alt="Screenshot 2024-10-25 at 2 33 58 PM" src="https://github.com/user-attachments/assets/8fee507e-0a7d-4405-ae65-60bb8bb610ed">
<img width="194" alt="Screenshot 2024-10-25 at 2 33 58 PM" src="https://github.com/user-attachments/assets/51462043-067a-43ca-8db3-216c4b7ef50b">
<img width="232" alt="Screenshot 2024-10-25 at 2 33 22 PM" src="https://github.com/user-attachments/assets/95a08b2a-89c5-454a-8705-56b22d94cc63">